CCI_indicator_view.sql

```diff
@@ -0,0 +1,2120 @@
+        , cte_cci_authorizer_sy22.prep_cte_pct AS prep_cte_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_ag_cte_pct AS prep_ag_cte_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_one_college_course_pct AS prep_one_college_course_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_two_college_course_pct AS prep_two_college_course_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_one_milsci_course_pct AS prep_one_milsci_course_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_two_milsci_course_pct AS prep_two_milsci_course_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_non_reg_pre_pct AS prep_non_reg_pre_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_reg_pre_pct AS prep_reg_pre_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_statefedjobs_pct AS prep_authorizerfedjobs_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_ssb_pct AS prep_ssb_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_tcbwe_pct AS prep_tcbwe_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_twbe_pct AS prep_twbe_pct_authorizer
+        , cte_cci_authorizer_sy22.prep_tcbwe_and_twbe_pct AS prep_tcbwe_and_twbe_pct_authorizer
+        , cte_cci_nonstate_sy22.prep_pct - cte_cci_authorizer_sy22.prep_pct AS prep_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_ap_pct - cte_cci_authorizer_sy22.prep_ap_pct AS prep_ap_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_ib_pct - cte_cci_authorizer_sy22.prep_ib_pct AS prep_ib_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_ag_pct - cte_cci_authorizer_sy22.prep_ag_pct AS prep_ag_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_cte_pct - cte_cci_authorizer_sy22.prep_cte_pct AS prep_cte_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_ag_cte_pct - cte_cci_authorizer_sy22.prep_ag_cte_pct AS prep_ag_cte_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_one_college_course_pct - cte_cci_authorizer_sy22.prep_one_college_course_pct AS prep_one_college_course_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_two_college_course_pct - cte_cci_authorizer_sy22.prep_two_college_course_pct AS prep_two_college_course_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_one_milsci_course_pct - cte_cci_authorizer_sy22.prep_one_milsci_course_pct AS prep_one_milsci_course_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_two_milsci_course_pct - cte_cci_authorizer_sy22.prep_two_milsci_course_pct AS prep_two_milsci_course_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_non_reg_pre_pct - cte_cci_authorizer_sy22.prep_non_reg_pre_pct AS prep_non_reg_pre_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_reg_pre_pct - cte_cci_authorizer_sy22.prep_reg_pre_pct AS prep_reg_pre_comparison_authorizer
+        , cte_cci_nonstate_sy22.prep_statefedjobs_pct - cte_cci_authorizer_sy22.prep_statefedjobs_pct AS prep_authorizerfedjobs_comparison_authorizer
```

Comment:
at what point do the comparison columns become kind of overkill? are all of these looked at separetly or is it just the overall status comparison we care about?